src/hotspot/share/opto/matcher.cpp

```diff
@@ -2252,7 +2252,6 @@ bool Matcher::find_shared_visit(MStack& mstack, Node* n, uint opcode, bool& mem_
     case Op_FmaVD:
     case Op_FmaVF:
     case Op_MacroLogicV:
-    case Op_LoadVectorMasked:
```

Comment:
Thanks for looking at this PR! This is removed since we added two combined rules in the aarch64_sve.ad (line-2383) like:
```
match(Set dst (VectorLoadMask (LoadVectorMasked mem pg)));
```
Setting `Op_LoadVectorMasked` as a root in matcher will make such rules not be matched. I'm also curious why this op is  added here. Does it have any influence if I remove it? Thanks!

Reply:
@jatin-bhateja, could you please help to check the influence about removing this? Kindly know your feedback about this. Thanks so much!

Reply:
Thanks a lot for looking at this part!